content/_index.md

```diff
@@ -38,15 +39,15 @@ Another advantage of Platform as a Service is that it can rid a developer to mai
 Yet another benefit of PaaS is that it helps you avoid writing certain kinds of code that you don't want to write. For example, platforms usually have built-in load balancers and autoscaling capabilities so that the platform can launch additional app instances automatically - you don't have to write the code that does this anymore. This is a very complex topic though, and if you want to dive more into those aspects...
 > TODO: add a pointer to further reading about load balancers, autoscaler and microservice patterns...
   
-# Developer Workflow
+## Developer Workflow
 
 Software development is hard. There's no getting around the base level of complexity involved in building software that doesn't completely fall apart as soon as a user breathes on it.
 
 The goal of any platform is to simplify the mundane boring part of development to let you focus your energy on more interesting problems. 
```

Comment:
We need to decide what to do with this section. I'd suggest a simple "manual" workflow without CI/CD and a remote repository. Something everybody can implement without additional tooling.

Reply:
Let's chat later this week about it.